pom.xml

```diff
@@ -293,6 +327,15 @@
                 <enabled>false</enabled>
             </releases>
         </repository>
+        <repository>
+            <id>central</id>
```

Comment:
I don't think we need specify this repository explicitly in POM. Can you please check and remove.